app/src/main/java/com/example/tasky/feature_agenda/presentation/event_detail_screen/EventDetailViewModel.kt

```diff
@@ -27,13 +46,22 @@ class EventDetailViewModel @Inject constructor(
     private val useCases: AgendaUseCases,
     private val repositories: AgendaRepositories,
     private val savedStateHandle: SavedStateHandle,
-    private val userPreferences: UserPreferences
+    private val userPreferences: UserPreferences,
+    private val userDataValidator: UserDataValidator
 ): ViewModel() {
 
     private val _state = MutableStateFlow(EventDetailState())
     val state = _state.asStateFlow()
 
+    private val resultChannel = Channel<Result<Unit>>()
+    val validationResult = resultChannel.receiveAsFlow()
+
     private val photoList = mutableStateListOf<Photo>()
+    var attendeeList = mutableStateListOf<Attendee>()
+        private set
```

Comment:
Can be a `val`. You won't replace the whole list, so it doesn't need to be a mutable field. The list itself will stay mutable, even as a `val`